repoman/pym/repoman/modules/scan/depend/_depend_checks.py

```diff
@@ -160,7 +161,12 @@ def _depend_checks(ebuild, pkg, portdb, qatracker, repo_metadata):
 			check_slotop(mydepstr, pkg.iuse.is_valid_flag,
 				badsyntax, mytype, qatracker, ebuild.relative_path)
 
+	dedup = collections.defaultdict(set)
 	for m, b in badsyntax:
+		if b in dedup[m]:
```

Comment:
@EvaSDK it's not useless, as is solves an issue reported in [bug 591184, comment #4](https://bugs.gentoo.org/show_bug.cgi?id=591184#c4). Of course there are multiple ways to use the set could be used to fix the problem. The way the way that I've done it here has the advantage of leaving the surrounding code unchanged.